.gitignore

```diff
@@ -1,4 +1,6 @@
 bin/
 gen/
 obj/
+ovr_mobile_sdk/
```

Comment:
> I am not sure if we should add "ovr_mobile_sdk" as gitignore. It can be a convenience but it seems a little weird to me.

Weird in what way? It certainly seems sensible to me - we require `ovr_mobile_sdk`, then have to keep ignoring its presence in `git status` reports.